src/features/vault/hooks/useFilteredPools.js

```diff
@@ -67,9 +67,9 @@ function showBoosted(pools) {
   });
 }
 
-function hideDecomissioned(pools) {
+function hideDecomissioned(pools, tokens) {
   return pools.filter(pool => {
-    return pool.status !== 'eol' && pool.status !== 'refund';
+    return (pool.status !== 'eol' && pool.status !== 'refund') || (tokens[pool.earnedToken] && tokens[pool.earnedToken].tokenBalance > 0);
```

Comment:
Should this be checking the `token` instead of the `earnedToken`?

Reply:
Oh, sorry, I was looking at the wrong code - was looking at the boost pools. Everything looks good to me, but I can't test either.